### PR TITLE
Added low watermark metric

### PR DESCRIPTION
**Pull request by rdonnelly3**
This PR is to address issue #9.

### CODE REVIEW

File: prometheus_kafka_consumer_group_exporter/__init__.py [hunk 1]
```diff
@@ -261,6 +265,29 @@ def update_highwater(offsets):
...
+                                  {'partition': partition, 'topic': topic})
+
+                    update_gauge(
+                        metric_name='kafka_topic_lowwater',
+                        label_dict={
+                            'topic': topic,
+                            'partition': partition
+                        },
+                        value=offsets[0],
+                        doc='The offset of the head of a partition in a topic.'
```
Comment (braedon) [inline prometheus_kafka_consumer_group_exporter/__init__.py]: Looks like the doc hasn't been changed from the high watermark metric. Perhaps:
```
The offset of the tail of a partition in a topic.
```

**Comment by braedon** [comment]
Merging to avoid conflict with implementation of https://github.com/braedon/prometheus-kafka-consumer-group-exporter/issues/8. Will do the doc/README changes myself once merged.

**Comment by rdonnelly3** [comment]
Thanks @braedon , sorry I haven't gotten to changing your suggested edits.  It was on my task list for this week.

**Comment by braedon** [comment]
All good!

**Pull request merged**